code/datums/components/mood.dm

```diff
@@ -4,6 +4,11 @@
 /datum/component/mood
 	var/mood //Real happiness
 	var/sanity = SANITY_NEUTRAL //Current sanity
+	var/list/datum/brain_trauma/psychological/disorders = list(	/datum/brain_trauma/psychological/depression,/datum/brain_trauma/psychological/schizophrenia/paranoid,
```

Comment:
Any reason to not use `subtypesof(/datum/brain_trauma/psychological)`?

Reply:
base schizophrenia, but i might just delet it